hooks/post_gen_project.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import os
+
+PROJECT_DIRECTORY = os.path.realpath(os.path.curdir)
+
+
+def remove_file(filepath):
+    os.remove(os.path.join(PROJECT_DIRECTORY, filepath))
+
+
+if __name__ == '__main__':
+    if '{{ cookiecutter.use_pypi_deployment_with_travis }}'.lower() == 'n':
```

Comment:
If `'{{ cookiecutter.use_pypi_deployment_with_travis }}'.lower() == 'y':`, then this hook will break the building of the package because `if __name__ == '__main__':`'s code block will lack even a pass.